test/storage/bucket.js

```diff
@@ -118,15 +118,19 @@ describe('Bucket', function() {
     var file;
     var metadata = { a: 'b' };
 
-    beforeEach(function() {
-      file = bucket.file(FILE_NAME, metadata);
+    it('should throw if no name was provided', function() {
+      assert.throws(function() {
+        file = bucket.file();
```

Comment:
This test would fail under the current state of the project. Not sure what is meant to be checked here.